leptos/src/lib.rs

```diff
@@ -140,6 +118,37 @@
 //! }
 //! # }
 //! ```
+//!
+//! # All Features
+//!
+//! | Feature                   | Description                                                   | Default |
+//! |:----------------------    |:------------------------------------------------------------  |:--------|
+//! | nightly                   | **Syntax**: conveniences like `name()` vs. `name.get()`       | No      |
+//! | csr[^1]                   | **Rendering**: Client-side. DOM nodes in browser.             | No      |
+//! | ssr[^1]                   | **Rendering**: Server-side. HTML typically on server.         | No      |
+//! | experimental-islands[^1]  | **Rendering**: Render components on server by default.        | No      |
+//! | hydrate[^1]               | **Rendering-ish**: Add interactivity to an SSR app.           | No      |
+//! | serde                     | **(De)Serialization**: use [serde] for [Resource]s.           | Yes     |
+//! | serde-lite                | **(De)Serialization**: use [serde-lite] for [Resource]s.      | No      |
+//! | rkyv                      | **(De)Serialization**: use [rkyv] for [Resource]s.            | No      |
+//! | miniserde                 | **(De)Serialization**: use [miniserde] for [Resource]s.       | No      |
+//! | tracing                   | **Logs & Telemetry**, [tracing] support for [component]s.     | No      |
+//! | default-tls[^2]           | **Networking**: Use default native TLS support.               | No      |
+//! | rustls[^2]                | **Networking**: Use [rustls].                                 | No      |
+//! | template_macro            | **Templating**: Enables the legacy [template] macro.          | No      |
+//!
+//! [^1]: At least one **rendering** feature must be enabled. An Isomorphic crate may have
```

Comment:
This rewritten version sounds misleading to me. Exactly one should be enabled, not more than one.